### ISSUE TITLE
Clarify help message for '-H'?

### ISSUE THREAD

**Issue by jchia** [itchyny/qhs#2]
I use qhs once in a blue moon, but I think when I used it, I often forget to specify `-H` because the description "skip the header row" doesn't seem like anything to me.

In my mind, nothing is being skipped -- the first row is used for either column names or row data.

Would it be clearer to change the help description from "Skip the header row" to "Treat first input line as header containing column names."? We can even change the long option from '--skip-header' to '--input-header', mirroring '--output-header'.

If it sounds good, I can send a PR.

**Comment by itchyny** [comment]
Note that qhs command is designed to work as same as the original [q command](http://harelba.github.io/q/) so I can't change the option name. But suggestion for changing the help description is welcome.
```
 % q -h | grep -A2 -- skip-header
    -H, --skip-header   Skip header row. This has been changed from earlier
                        version - Only one header row is supported, and the
                        header row is used for column naming
```
The original command says `Skip header row`.

**Comment by jchia** [comment]
I didn't realize qhs is a Haskell port/parallel of the Python q. I see the help message in the Python q is currently as you pointed out, and it alludes to "column names". I assume "earlier version" is irrelevant to qhs.

Maybe we can use "Skip header row for row input and use it for column names instead" to avoid changing things too much.

**Comment by itchyny** [comment]
How about `Use the first row as column names.`? Assuming the row as the column names, skipping it as data is obvious.

**Comment by jchia** [comment]
Sure, but I thought some users might be left scratching their head about the meaning of 'skip' in `--skip-header` without further explanation. Personally, I didn't think of this option as skipping anything at all but just using the first row as a proper header and "skip the first line for row input" is how I made-sense-of/rationalized it.

**Comment by itchyny** [comment]
Released as [v0.3.2](https://github.com/itchyny/qhs/releases/tag/v0.3.2).
